consensus/src/test/kotlin/maru/consensus/ProtocolStarterTest.kt

```diff
@@ -72,9 +78,12 @@ class ProtocolStarterTest {
       createProtocolStarter(
         forksSchedule = forksSchedule,
         clockMilliseconds = 16000, // After fork transition at 15
+        forkTransitionNotifier = forkTransitionNotifier,
       )
     protocolStarter.start()
     assertActiveProtocol2(protocolStarter)
+    assertThat(forkTransitions.size).isEqualTo(1)
+    assertThat(forkTransitions.first()).isEqualTo(forkSpec2)
```

Comment:
Simple assertions like this, imo, should be done as follow:
```suggestion
    assertThat(forkTransitions).isEqualTo(listOf(forkSpec2))
```

1. easier to read/understand
2. when it fails, having the list of items is easier to see what is missing or should not be in the list.